modules/openapi-generator/src/main/resources/typescript/services/ObjectParamAPI.mustache

```diff
@@ -1,5 +1,5 @@
 import type { HttpFile } from '../http/http';
-import type { Configuration } from '../configuration'
+import type { Configuration } from '../configuration';
 import type * as req from "../types/ObjectParamAPI";
```

Comment:
I did notice that different templates use different quotes to import - it's currently:

* 52x single quoted imports
* 70x double quoted imports

in the `typescript` generator. Is there a guidance on what to use? If yes, I'd unify.

Reply:
good question
probably we should just use single quotes everywhere